src/components/panels/Mmu/MmuFilamentStatus.vue

```diff
@@ -325,6 +314,22 @@ export default class MmuFilamentStatus extends Mixins(BaseMixin, MmuMixin) {
         return POSITIONS.UNKNOWN
     }
 
+    get endOfBowdenPos() {
+        if (typeof this.toolheadSensor === 'boolean' && !this.configExtruderForceHoming) return POSITIONS.END_BOWDEN
+
+        if (
+            this.configExtruderHomingEndstop === 'none' ||
+            this.configExtruderHomingEndstop === 'collision' ||
+            this.configExtruderHomingEndstop === 'mmu_gear_touch' ||
+            this.configExtruderHomingEndstop === 'filament_compression'
+        )
+            return POSITIONS.EXTRUDER_ENTRANCE
```

Comment:
```suggestion
        const homingEndstops = ['none', 'collision', 'mmu_gear_touch', 'filament_compression']
        if (homingEndstops.includes(this.configExtruderHomingEndstop)) return POSITIONS.EXTRUDER_ENTRANCE
```

I would write it this way. its a little bit better readable. Pls correct the name homingEndstops, if you know a better name for the array of names.